packages/apify/src/apify.ts

```diff
@@ -532,15 +529,14 @@ export class Apify {
      * For more information, see  {@link Apify.openKeyValueStore}
      * and  {@link KeyValueStore.getValue}.
      *
-     * @param {string} key
-     *   Unique record key.
-     * @returns {Promise<Object<string, *>|string|Buffer|null>}
+     * @param key Unique record key.
+     * @returns
      *   Returns a promise that resolves to an object, string
      *   or [`Buffer`](https://nodejs.org/api/buffer.html), depending
      *   on the MIME content type of the record, or `null`
      *   if the record is missing.
      */
-    async getValue(key) {
+    async getValue(key: string): Promise<Record<string, unknown> | string | Buffer | null> {
```

Comment:
we should add generics here, cc @vladfrangu

Reply:
We can probably copy the generics from the KeyValue store and put them here as well, right?

Reply:
yop